src/platform/platform_internal.h

```diff
@@ -919,6 +922,8 @@ typedef struct CXPLAT_SOCKET {
 
     uint8_t UseTcp : 1;                  // Quic over TCP
 
+    uint8_t IsServer : 1;
```

Comment:
`HasFixedRemoteAddress` already is effectively conveys this state. If there is no fixed remote address, then it's a server (unconnected) socket. If there is a fixed remote address, then it's a client socket.

Reply:
Updated.